swap/swap_test.go

```diff
@@ -761,7 +741,7 @@ func TestPeerVerifyChequeProperties(t *testing.T) {
 
 // TestPeerVerifyChequeProperties tests that verifyChequeProperties will reject invalid cheques
 func TestPeerVerifyChequePropertiesInvalidCheque(t *testing.T) {
-	swap, peer, dir := newTestSwapAndPeer(t)
+	_, peer, dir := newTestSwapAndPeer(t)
```

Comment:
I would prefer to keep `swap` and then use `swap.owner.privateKey` when signing, avoids reliance on global variables and makes it clearer IMHO. What do you think?

Reply:
`swap.owner.privateKey` is actually the private key for `beneficiaryAddress` (if it were set correctly). But we want to sign with `ownerKey`.